include/rocksdb/transaction_log.h

```diff
@@ -113,7 +113,13 @@ class TransactionLogIterator {
     // Default: true
     bool verify_checksums_;
 
-    ReadOptions() : verify_checksums_(true) {}
+    // if true, the mapping of db sequence to WAL file block index will be
+    // cached. This prevents repeated reading from the beginning of the
+    // target wal log when GetUpdatesSince() is called.
+    // Default: true
```

Comment:
We should not enable this by default since this won't work with WAL compression, where the first record sets the compression type.

Reply:
👌